liquibase-standard/src/test/groovy/liquibase/change/core/CreateTableChangeTest.groovy

```diff
@@ -324,4 +333,30 @@ public class CreateTableChangeTest extends StandardChangeTest {
         assertTrue(statement.getColumnRemarks("id").equals("test remark"))
 
     }
+
+    def "generateStatementsWithIfNotExists"() {
+        when:
+        def change = new CreateTableChange(tableName: "test_table")
+        def columnConfig = new ColumnConfig()
+        columnConfig.setName("id")
+        columnConfig.setType(type)
+        change.addColumn(columnConfig)
+        change.setIfNotExists(true)
+
+        then:
+        SqlGeneratorFactory.getInstance().generateSql(change, database)[0].toSql() == expected
+
+        where:
+        type          | autoinc | database               | expected
```

Comment:
I believe we should add test cases for the databases that do **not** support `CREATE TABLE ... IF NOT EXISTS` just to ensure that the `IF NOT EXISTS` does not get added to the generated SQL unexpectedly.

CC @rberezen @filipelautert